AddDocs.js

```diff
@@ -0,0 +1,39 @@
+const fs = require('fs');
+const DiscoveryV1 = require('ibm-watson/discovery/v1');
+const { IamAuthenticator } = require('ibm-watson/auth');
+
+const discovery = new DiscoveryV1({
+  version: '2019-04-30',
+  authenticator: new IamAuthenticator({
+    apikey: 'CMfoTLs3yHLDTVBqCWnhbJntlj3io-Ojg4QdEdGfuKQ6',
+  }),
+  url: 'https://api.us-south.discovery.watson.cloud.ibm.com/instances/3b5bd372-ed9b-4bbb-bce9-a93526329807',
+});
+
+const addDocumentParams = {
+  environmentId: '32d99835-9310-4873-a29d-b2be6aaa1fad',
+  collectionId: 'aabcf357-e6dd-4487-aa2b-d2bfc65876c0',
+  file: fs.createReadStream('./test-doc4.html'),
+};
+
+discovery.addDocument(addDocumentParams)
+  .then(response => {
+    const documentAccepted = response.result;
+    console.log(JSON.stringify(documentAccepted, null, 2));
+  })
+  .catch(err => {
+    console.log('error:', err);
+  });
+
+  const queryParams = {
+    environmentId: '32d99835-9310-4873-a29d-b2be6aaa1fad',
```

Comment:
save environemnt id in a variable, so that you don't have to paste it everywhere. You can also store it in `env` file.

Reply:
can this environment ID be saved in the same env file I added the credentials to?

Reply:
So looks like I can add the environment ID in the same env file but I haven't figure out how to add it to the file. Is there documentation on how the environment ID should be added in the file? I've tried using the same format as when adding credentials but it's not working. Either that or something in the code is not right.

Reply:
I didn’t go the `dotenv npm` package way as I found a way to add the `environmentId` it in the existing .env file and run the existing code without having to use the dotenv npm package. Since I was able to run it with the environmentID I also went ahead and added the `collectionId` to the same file which also worked.

I will upload the new code changes